### DISCUSSION TITLE
An improvement for the Load Additional Data tab under Other emwin

### DISCUSSION THREAD

**Discussion by stonegrift** [JVital2013/vitality-goes#75] [Ideas]
The amount of available data from NOAA is daunting to deal with.  This idea is for an improvement of either a mouse over pop-up or a replacement of the Product section. Using the list at https://w2.weather.gov/source/datamgmt/xr04_X_ref_by_NNN.html replace or add to the 3 letter codes the actual name of each product. So it is easier to choose.  Personally I end up filtering right to left starting with State then originator and then the product since the other two impact the available products.  Open for discussion since php is a bit beyond my skill level at the moment.

**Comment by JVital2013** [comment]
This is a very good idea indeed. I'll probably scrape this and include it as a part of Vitality GOES to avoid the internet dependency. From there, it can be used as a lookup table for loading additional data.

Opened issue #76 to track this feature request.
